dydxV4/dydxV4/_Tracking/TrackingViewController+Ext.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  TrackingViewController+Ext.swift
```

Comment:
this files serves the purpose of creating paths to identify the screen based on RoutingRequest data.